src/data/veilederinfo/types/Veileder.ts

```diff
@@ -0,0 +1,11 @@
+export interface Veileder {
+  ident: string;
+  fornavn: string;
+  etternavn: string;
+  epost: string;
+  telefonnummer?: string;
+}
```

Comment:
Kunne godt kalt det `VeilederDTO` for min del, for å vise at dette kommer fra en annen app og at hvis man endrer noe her så er det breaking.

Reply:
Hm, det er godt poeng. Kan legge på `DTO`